docs/feature-gates.md

```diff
@@ -55,6 +56,19 @@ manifest provided as part of releases enables this feature by default. If you
 edit the manifest, make sure you do not disable it, as it is needed for correct
 NetworkPolicy implementation for Pod-to-Service traffic.
 
+### EndpointSlice
+
+`EndpointSlice` enables Service EndpointSlice support in AntreaProxy. The
+EndpointSlice API was introduced in Kubernetes 1.16 (alpha) and it is enabled
+by default in Kubernetes 1.17 (beta). This flag will take no effect if AntreaProxy
```

Comment:
Probably clearer to replace "This flag" with "The EndpointSlice feature gate".

Reply:
Thanks for advice, I will handle today.